From 3f42d9d6f3d9c4408b6118f12d59bd6fe0f4473b Mon Sep 17 00:00:00 2001
From: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
Date: Wed, 9 Aug 2023 23:36:24 +0600
Subject: [PATCH 91/92] arm64: dts: renesas: vc4: use
 "ethernet-phy-ieee802.3-c45" for mv88q3344_es

This is required for proper c45 PHY initialization.

Signed-off-by: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a779f0-vc4-tsn0-phy-2g5.dtso | 2 +-
 arch/arm64/boot/dts/renesas/r8a779f0-vc4-tsn1-phy-2g5.dtso | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a779f0-vc4-tsn0-phy-2g5.dtso b/arch/arm64/boot/dts/renesas/r8a779f0-vc4-tsn0-phy-2g5.dtso
index 3dd7ffc34f35..b1b53f16fe06 100644
--- a/arch/arm64/boot/dts/renesas/r8a779f0-vc4-tsn0-phy-2g5.dtso
+++ b/arch/arm64/boot/dts/renesas/r8a779f0-vc4-tsn0-phy-2g5.dtso
@@ -28,7 +28,7 @@
 
 	tsn0_phy_2g5: ethernet-phy@0 {
 		reg = <0>;
-		compatible = "ethernet-phy-id0000.0000";
+		compatible = "ethernet-phy-ieee802.3-c45";
 		status = "okay";
 	};
 };
diff --git a/arch/arm64/boot/dts/renesas/r8a779f0-vc4-tsn1-phy-2g5.dtso b/arch/arm64/boot/dts/renesas/r8a779f0-vc4-tsn1-phy-2g5.dtso
index e4233872a880..4d67a7932352 100644
--- a/arch/arm64/boot/dts/renesas/r8a779f0-vc4-tsn1-phy-2g5.dtso
+++ b/arch/arm64/boot/dts/renesas/r8a779f0-vc4-tsn1-phy-2g5.dtso
@@ -21,7 +21,7 @@
 
 	tsn1_phy_2g5: ethernet-phy@0 {
 		reg = <0>;
-		compatible = "ethernet-phy-id0000.0000";
+		compatible = "ethernet-phy-ieee802.3-c45";
 		status = "okay";
 	};
 };
-- 
2.30.2

